### PR TITLE
fix: add missing options for expiry settings

### PR DESCRIPTION
**Pull request by MrDeerly**
Adds two missing options to the expiration settings. 

I also renamed the page because it doesn't just cover ID Tokens.

### CODE REVIEW

File: content/docs/tokens.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ __NOTE__: All duration options should be set in the format: number + time unit (
 
 * `expiry` - section for various expiration settings, including token settings:
   * `idTokens` - the lifetime of id_token. It is preferable to use short-lived id tokens.
+  * `deviceRequests`: the lifetime of device_requests.
+  * `signingKeys`: the lifetime of signing_token.
```
Comment (nabokihms) [inline content/docs/tokens.md:74]:
```suggestion
  * `deviceRequests` - the lifetime of device_requests.
  * `signingKeys` - the lifetime of signing_token.
```

**Review by nabokihms — CHANGES_REQUESTED** [review]
A small NIT was left to be in line with other options. Besides it, LGTM.
image

**Comment by MrDeerly** [comment]
> A small NIT was left to be in line with other options. Besides it, LGTM. image

Good Catch!

**Review by nabokihms — APPROVED** [review]

**Comment by nabokihms** [comment]
:+1:

**Pull request merged**